ui-modules/blueprint-composer/app/components/util/model/entity.model.js

```diff
@@ -115,6 +116,8 @@ export class Entity {
      *
      */
     touch() {
+        // include a summary to aid with debugging (otherwise log just shows the property lastUpdated)
+        this.summary = (this.type || "unset") + (this.id ? " "+this.id : "");
```

Comment:
I don't get the idea behind the summary, we already the type and id within the object. Can you explain this please @ahgittin ?

Reply:
without this when we `console.log(entity)` we can't identify different entities, and it looks weird, as it just shows `Entity { lastUpdated: xxxx }`.

Reply:
okay i'll comment out and we can re-enable if it's wanted for debugging